### PR TITLE
chore(table): clean up demo pages

### PR DESCRIPTION
**Pull request by andrewseguin**
The main demo page contains too much - reduced to be simpler and added a few more pages for different features

### CODE REVIEW

File: src/demo-app/table/custom-table/custom-table.scss [hunk 1]
```diff
@@ -1,8 +1,9 @@
-.mat-table {
-  height: 300px;
-  overflow: auto;
+.mat-table, wrapper-table {
+  width: 100%;
 }
 
 .mat-card {
   margin: 8px 0;
+  height: 300px;
```
Comment (josephperrott) [inline src/demo-app/table/custom-table/custom-table.scss]: alphabetical order.

Comment (jelbourn) [reply src/demo-app/table/custom-table/custom-table.scss]: We don't have a rule for css property order, actually

**Comment by andrewseguin** [comment]
Caretaker: Will require BUILD changes, see change 193566542

**Comment by mmalerba** [comment]
@andrewseguin there seems to be an issue with the AOT test

**Pull request merged**